src/item/component/RenderOffsetsComponent.php

```diff
@@ -3,40 +3,46 @@
 
 namespace customiesdevs\customies\item\component;
 
-final class RenderOffsetsComponent implements ItemComponent {
 
-	private int $textureWidth;
-	private int $textureHeight;
-	private bool $handEquipped;
+final class RenderOffsetsComponent implements ItemComponent { //thanks to https://github.com/MedicalJewel105/bedrock-render-offsets-generator/blob/main/main.py
```

Comment:
If you want to keep this credits comment, please put it with the `getValue()` method and use proper PHP Doc styling